kolibri/plugins/device/assets/src/views/DeprecationWarningBanner.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+
+  <div
+    v-show="showBanner"
+    class="alert"
+    :style="{ backgroundColor: $themePalette.yellow.v_100 }"
+  >
+    <div style="display:flex">
+      <div>
+        <KIcon
+          icon="warning"
+          class="icon"
+          :color="$themePalette.amber.v_a400"
+        />
+      </div>
+
+      <div class="error-message">
+        <p v-if="py27Deprecated">
+          {{ coreString('pythonSupportWillBeDropped') }}
+        </p>
+        <p v-if="currentUserOnIE11">
```

Comment:
This is showing both texts in the case that both apply:
![image](https://github.com/learningequality/kolibri/assets/1680573/e6a00475-6ed9-47c5-8b55-7df69f0f3928)

Although, I think it reads OK.